samples/unit-tests/series-networkgraph/networkgraph/demo.js

```diff
@@ -88,6 +99,16 @@ QUnit.test('Network Graph', function (assert) {
         'Custom series.data.dashStyle (#9798)'
     );
 
+    chart.series[0].nodes[0].setState('select');
+
+    assert.strictEqual(
+        chart.series[0].nodes.filter(
+            node => node.graphic.element.getAttribute('fill') === 'orange'
+        ).length,
+        1,
+        'Only one node has selected attributes (#11212)'
```

Comment:
Better (but not a blocker): `'Only one node should have selected attributes (#11212)'`.